dashboard/cron.py

```diff
@@ -375,9 +375,10 @@ def do(self):
 
         status += self.submission()
         status += self.weight_consideration()
-        
-        logger.info("************ informational")
-        status += self.update_unizin_metadata()
+
+        if settings.DATA_WAREHOUSE_IS_UNIZIN:
```

Comment:
if the DATA_WAREHOUSE_IS_UNIZIN is false, then there is no metadata shown at all?

Reply:
Yes as per #349 the metadata table seems to be a Unizin table (not part of Canvas Data). from what I understand (unless I missed something) the metadata is only being used for displaying the last updated time to the user currently. When `DATA_WAREHOUSE_IS_UNIZIN` is false, I switched it to show the last time the cron job ran (which was what was used before switching to Unizin metadata).